docs/AllRectorsOverview.md

```diff
@@ -291,7 +291,7 @@ Turns magic callback assign to callback assign on Nette Form events.
 ```diff
 -$someService = $container->someService;
 +$container->getService("someService");
- 
```

Comment:
@TomasVotruba These spaces are coming from nowhere :scream:

Reply:
@carusogabriel clean_levels.sh is guilty here:D

Reply:
They're comming from the `DescribeCommand`.

Check this method: https://github.com/rectorphp/rector/blob/baa8a09014492c93935d4242734acfc7207f3b4a/src/Console/Command/DescribeCommand.php#L167

Reply:
@mssimi No, because it is executed before `update-docs`.

@TomasVotruba Can't figure out this one. Gonna try later, of fell free to fix it

Reply:
@carusogabriel No worries, take you time.

Writing test for 1 RectorDefinition and expected no-extra-spaces output would be best to start with